doc/manual/metaprogramming.rst

```diff
@@ -620,11 +629,14 @@ Compare:
     Expr
       head: Symbol string
       args: Array{Any}((5,))
-        1: String "a ("
+        1: String
+          data: Array{UInt8}((3,)) UInt8[0x61,0x20,0x28]
```

Comment:
I agree the output isn't as nice. Would it be worth opening an issue to confirm the changes to String output was intentional?